### PR TITLE
USHIFT-1266: update configure-vm.sh to use 4.13 RPM repository for dependencies

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by dhellmann** [comment]
/hold 

There are no openshift-clients in the 4.13 RPM repo, yet.

File: scripts/devenv-builder/configure-vm.sh [hunk 1]
```diff
@@ -107,10 +107,13 @@ fi
...
     OSVERSION=$(awk -F: '{print $5}' /etc/system-release-cpe)
+    # This version might not match the version under development
+    # because we need to pull in dependencies that are already
+    # released.
+    OCPVERSION=4.13
     sudo subscription-manager config --rhsm.manage_repos=1
-    # TODO: Start using 'rhocp-4.13' repository when OCP 4.13 is released
     sudo subscription-manager repos \
-        --enable "rhocp-4.12-for-rhel-${OSVERSION}-$(uname -m)-rpms" \
+        --enable "rhocp-${OCPVERSION}-for-rhel-${OSVERSION}-$(uname -m)-rpms" \
```
Comment (ggiguash) [inline scripts/devenv-builder/configure-vm.sh:116]: We need to make a similar change in the build.sh file and the devenv docs.

**Comment by dhellmann** [comment]
I think we are likely to want to take #1818 instead of this PR.

**Comment by dhellmann** [comment]
/close

Let's take #1818 instead

**Pull request closed without merging**